jujugui/static/gui/src/app/components/search-results/test-search-results.js

```diff
@@ -25,12 +25,18 @@ chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
 describe('SearchResults', function() {
+  var searchResults;
 
   beforeAll(function(done) {
-    // By loading this file it adds the component to the juju components.
+    // By loading these files it makes their classes available in the tests.
     YUI().use('search-results', function() { done(); });
   });
 
+  beforeEach(function() {
+    searchResults = new juju.components.SearchResults();
```

Comment:
You're mixing the extension tests with the component tests. I would like to keep these component tests restricted to only testing the public api's and output of the components and no (or very few) unit tests.